frontend/src/lib/lemon-ui/LemonButton/LemonButton.tsx

```diff
@@ -75,6 +75,7 @@ export type SideAction = Pick<
     LemonButtonProps,
     | 'onClick'
     | 'to'
+    | 'loading'
```

Comment:
This feels like a mistake?

Reply:
ah interesting. I couldn't find it in the code usage but if you're sure then all good